packages/kmod-6.1-nvidia-r570/grid-license-check.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=GRID License Check
+RefuseManualStart=true
+RefuseManualStop=true
+DefaultDependencies=no
+Before=kubelet.service
+After=nvidia-gridd.service
+Requires=nvidia-gridd.service
+
+[Service]
+Type=oneshot
+ExecCondition=/usr/bin/ghostdog match-nvidia-driver grid
+# Otherwise, attempt to load the module.
+ExecStart=/usr/bin/nvidia-smi -q
+# Ensure that the stderr file exists. Otherwise, grep fails on an empty file.
+ExecStart=-/usr/bin/touch /tmp/.nvidia-gridd-license
```

Comment:
`truncate` may be better than `touch` to reset the file between iterations

(would have to move this before the `nvidia-smi` call)

Reply:
I like that, I'll give it a shot to see if it still gives me the behavior I want with things rearranged to use `truncate`